src/lib.rs

```diff
@@ -241,21 +248,25 @@ mod hwb;
 
 mod hues;
 
-mod tristimulus;
 mod convert;
 mod equality;
+///Provides the Chromatic Adpatation to convert color from one illuminant to another
+pub mod chromatic_adaptation;
+///Defines the tristimulus values of the CIE Illuminants..
```

Comment:
I would appreciate it if these comments can be moved into the modules. That keeps everything in the same place.